frontend/packages/data-portal/app/components/Filters/IdFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from 'react'
+
+import { QueryParams } from 'app/constants/query'
+
+import { RegexFilter } from './RegexFilter'
+
+export function IdFilter({
```

Comment:
since this looks like it's supposed to be filtering CryoET entity prefixes specifically, maybe name it EntityIdFilter or something? 🤷